common/decklist.cpp

```diff
@@ -153,6 +156,17 @@ AbstractDecklistNode *InnerDecklistNode::findChild(const QString &_name)
     return nullptr;
 }
 
+AbstractDecklistNode *InnerDecklistNode::findCardChildByNameAndProviderId(const QString &_name,
+                                                                          const QString &_providerId)
+{
+    for (int i = 0; i < size(); i++) {
+        if (at(i) != nullptr && at(i)->getName() == _name && at(i)->getCardProviderId() == _providerId) {
```

Comment:
NIT: @ebbit1q Does calling `at(i)` 3 times here cause any potential issues? Would it make sense to store as a temporary?